src/libraries/System.Security.Cryptography/tests/HmacSha1Tests.cs

```diff
@@ -39,7 +39,7 @@ public HmacSha1Tests()
         }
 
         protected override int BlockSize => 64;
-        protected override int MacSize => 20;
+        protected override int MacSize => HMACSHA1.HashSizeInBytes;
```

Comment:
Personally, I would have left these as magic numbers of "what's the right answer?" vs "well, the const was wrong, so it produced the wrong sized output... but it did so faithfully!"; but I don't feel strongly.

It's a slight bit weird that for HMACMD5 you used the MD5 (non-HMAC) const, but for the SHA ones you used the HMACSHA* ones.

But, again, it merely falls in the "worth commenting on" zone that's not quite at "I have a real opinion here".

Reply:
> It's a slight bit weird that for HMACMD5 you used the MD5 (non-HMAC) const

That is a mistake and will bug me forever, so I fixed it. Though it looks like I raced you in a merge.